frontend/src/app/main/ui/viewer.cljs

```diff
@@ -116,7 +117,9 @@
          (mf/deps overlay close-click-outside?)
          (fn [_]
            (when close-click-outside?
-             (st/emit! (dv/close-overlay (:id overlay))))))]
+             (if (:animation overlay)
```

Comment:
good place to use `(if-let`  (not very important but anyway...)

Reply:
I need more info about this, I have change it but I don't know the differents between them :)